custom_components/easee/sensor.py

```diff
@@ -8,13 +8,16 @@
 import logging
 
 from easee import Charger, ChargerState, ChargerConfig, Site, Circuit
+from easee.charger import ChargerSchedule
+from easee.utils import validate_iso8601
```

Comment:
Did you run `Make lint`? 
This function is not used

Reply:
Probably auto-imported when I was trying to get it to work. Will remove.